OmniDB/OmniDB_app/static/OmniDB_app/js/Connections.js

```diff
@@ -535,10 +535,21 @@ function showConnectionList() {
 
 				var col = new Object();
 				col.title =  'Group';
-				col.type = "checkbox",
-				col.checkedTemplate = true,
-        col.uncheckedTemplate = false
-				col.width = '60'
+				col.type = "checkbox";
+				col.checkedTemplate = true;
+        		col.uncheckedTemplate = false;
+				col.width = '60';
+				/*TODO: verificar isso (atualizar hot)
```

Comment:
@barthisrael , please keep all comments in English.

Reply:
Sure, @wind39 . That wasn't even supposed to be in the PR, sorry.